extensions/sendbird/v1/actions/deleteMetadata/deleteMetadata.ts

```diff
@@ -0,0 +1,82 @@
+import { z, ZodError } from 'zod'
+import { fromZodError } from 'zod-validation-error'
+import { type Action } from '@awell-health/extensions-core'
+import { type settings } from '../../../settings'
+import { Category, validate } from '@awell-health/extensions-core'
+import { SettingsValidationSchema } from '../../../settings'
+import { FieldsValidationSchema, fields } from './config'
+import {
+  SendbirdClient,
+  isSendbirdChatError,
+  sendbirdChatErrorToActivityEvent,
+} from '../../client'
+import { isEmpty } from 'lodash'
+
+export const deleteMetadata: Action<typeof fields, typeof settings> = {
+  key: 'deleteMetadata',
+  title: 'Delete metadata',
+  description: 'Deletes user metadata using Chat API.',
+  category: Category.COMMUNICATION,
+  fields,
+  previewable: false,
+  onActivityCreated: async (payload, onComplete, onError) => {
+    try {
+      const {
+        settings: { applicationId, chatApiToken, deskApiToken },
+        fields: { userId, metadataKey },
+      } = validate({
+        schema: z.object({
+          settings: SettingsValidationSchema,
+          fields: FieldsValidationSchema,
+        }),
+        payload,
+      })
+
+      const client = new SendbirdClient({
+        applicationId,
+        chatApiToken,
+        deskApiToken,
+      })
+
+      await client.chatApi.deleteMetadata(
+        userId,
+        !isEmpty(metadataKey) ? metadataKey : undefined
```

Comment:
According to the docs, this key should be URL encoded. Can you double-check that?
- https://sendbird.com/docs/chat/platform-api/v3/user/managing-metadata/user-delete-metadata
- https://sendbird.com/docs/chat/platform-api/v3/prepare-to-use-api#2-url-encoding

Reply:
It's already implemented in the client code for all endpoints that use `string` variables inside url